<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search_Radio_Form</class>
 <widget class="QDialog" name="Search_Radio_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Radio Image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="vImagelLay">
     <item>
      <layout class="QHBoxLayout" name="hLayoutImage">
       <item>
        <widget class="QLabel" name="lImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Image :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wImage" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hAzimuthlLay">
       <item>
        <widget class="QLabel" name="lAzimuth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Azimuth :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lEAzimuth"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hElevationLay">
       <item>
        <widget class="QLabel" name="lElevation">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Elevation :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lEElevationAngle">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="vButtonsLay">
     <item>
      <widget class="QToolButton" name="tbLoadImage">
       <property name="toolTip">
        <string>Load Image</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbClearImage">
       <property name="toolTip">
        <string>Clear Image</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="hButtonsLay">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>lImage</zorder>
  <zorder>wImage</zorder>
  <zorder>tbLoadImage</zorder>
  <zorder>tbClearImage</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>pbOk</zorder>
  <zorder>pbCancel</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>lAzimuth</zorder>
  <zorder>lEAzimuth</zorder>
  <zorder>lElevation</zorder>
  <zorder>lEElevationAngle</zorder>
  <zorder>lElevation</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
